app/components/subject-viewer.cjsx

```diff
@@ -79,20 +79,20 @@ module.exports = React.createClass
     {type, format, src} = getSubjectLocation @props.subject, @state.frame
     if @state.inFlipbookMode
       mainDisplay = @renderFrame @state.frame
+      console.log ' @state.frame',  @state.frame
     else
       mainDisplay = @props.subject.locations.map (frame, index) =>
         @renderFrame index, {key: "frame-#{index}"}
-
     tools = switch type
       when 'image'
         if not @state.inFlipbookMode or @props.subject?.locations.length < 2 or subjectHasMixedLocationTypes @props.subject
-          if @props.allowFlipbook and @props.allowSeparateFrames
+          if @props.workflow?.configuration.enable_switching_flipbook_and_separate
             <button className="secret-button" aria-label="Toggle flipbook mode" title="Toggle flipbook mode" onClick={@toggleInFlipbookMode}>
               <i className={"fa fa-fw " + if @state.inFlipbookMode then "fa-th-large" else "fa-film"}></i>
             </button>
         else
           <span className="tools">
-            {if @props.allowFlipbook and @props.allowSeparateFrames
+            {if @props.workflow?.configuration.enable_switching_flipbook_and_separate
```

Comment:
Might be something to do with whether the play controls get rendered, I'm not sure. Anyway, it's outside the scope of this PR but looks like a bit of code that could be tidied up to avoid duplication.